client/src/app/shared/loading-bar/loading-bar.component.html

```diff
@@ -0,0 +1,3 @@
+<ng-container *ngIf="_isVisible">
+  <mat-progress-bar color="accent" mode="indeterminate"></mat-progress-bar>
```

Comment:
Let's make those expressions and receive them from the component. In fact, they could be inputs to our component, so consumers could configure it the same way from the outside

```
<ml-progress-bar color="accent" mode="indeterminate"></ml-progress-bar>
```

Reply:
will do